lib/resources/dstu2/patient.yaml

```diff
@@ -27,15 +29,282 @@ fields:
+
+- name: active
+  required: 'No'
+  cardinality: 0..1
+  type: boolean
+  description: Whether this patient record is in active use.
+  action: create
+  example: |
+    {
+      "active": true
+    }
+  note: If set, active must be true
+  url: http://hl7.org/fhir/dstu2/patient-definitions.html#Patient.active
+
+- name: name
+  required: 'Yes'
+  cardinality: 1..*
+  type: HumanName
+  description: The names associated with the individual.
+  action: create
+  example: |
+    {
+      "name": [
+        {
+          "use": "official",
+          "family": [
+            "Wolf"
+          ],
+          "given": [
+            "Big",
+            "Bad"
+          ],
+          "period": {
+            "start": "2010-05-17T14:54:31.000Z"
+          }
+        },
+        {
+          "use": "usual",
+          "given": [
+            "Bigby"
+          ],
+          "period": {
+            "start": "2012-05-22T15:45:50.000Z"
+          }
+        }
+      ]
+    }
+  note: |
+    Must have one value that is current and has a `use` of 'official', without an end date, and both a given and family name are present.
+    Names must include the `use` field, and must contain either the `given` or `family` field. All names cannot contain the `text` field.
+    Only one `family` name. If additional family names are required they must be provided as a single string separated by blank spaces.
+    Upto two `given` names may be provided. If additional given names are required they must be provided in the second value as a single
+    string separated by blank spaces. Only one `prefix` and one `suffix` may be provided.
+  url: http://hl7.org/fhir/dstu2/patient-definitions.html#Patient.name
+
+- name: telecom
+  required: 'No'
+  cardinality: 0..*
+  type: ContactPoint
+  description: A contact detail (e.g. a telephone number or an email address) by which the individual may be contacted.
+  action: create
+  example: |
+    {
+      "telecom": [
+        {
+          "system": "phone",
+          "value": "8168229121",
+          "use": "home",
+          "period": {
+            "start": "2012-05-17T15:33:18.000Z"
+          }
+        }
+      ]
+    }
+  note: All values must contain the `system`, `use`, and `value` fields.
+  url: http://hl7.org/fhir/dstu2/patient-definitions.html#Patient.telecom
+
+- name: gender
+  required: 'No'
+  cardinality: 0..1
+  type: code
+  description: 'Administrative Gender - the gender that the patient is considered to have for administration and record keeping purposes.'
+  action: create
+  example: |
+    {
+      "gender": "male"
+    }
+  url: http://hl7.org/fhir/dstu2/patient-definitions.html#Patient.gender
+
+- name: birthDate
+  required: 'No'
+  cardinality: 0..1
+  type: date
+  description: The date of birth for the individual.
+  action: create
+  example: |
+    {
+      "birthDate": "1990-09-15"
+    }
+  note: birthDate must not contain a time component.
```

Comment:
I don't think we need this note: birthDate is defined as a "Date" field, it's impossible for it to contain a time (that would be DateTime or instant types) :)

Reply:
Removed the note